angularjs-portal-home/src/main/webapp/my-app/layout/partials/welcome.html

```diff
@@ -0,0 +1,8 @@
+<div class="welcome-popup center">
+  <h1>Welcome to the new MyUW. Here are the basics.</h1>
+  <img src="img/myuw-welcome.png">
```

Comment:
Equivalent-access accessibility was [discussed when this story was groomed](https://jira.doit.wisc.edu/jira/browse/MUMUP-1947?focusedCommentId=597978&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-597978), as I recall.